lib/view_component/slotable_v2.rb

```diff
@@ -103,15 +108,15 @@ def renders_one(slot_name, callable = nil)
       # = Setting sub-component content
       #
       # Consumers of the component can set the content of a slot by calling a
-      # helper method with the same name as the slot. The method can be
-      # called multiple times to append to the slot.
+      # helper method with the same name as the slot prefixed with `with`. The
```

Comment:
To be extra clear... ?

```suggestion
      # helper method with the same name as the slot prefixed with `with_`. The
```

Reply:
Good call, thank you!

Reply:
> To be extra clear... ?
> 
> ```suggestion
>       # helper method with the same name as the slot prefixed with `with_`. The
> ```